toasty/image.py

```diff
@@ -732,7 +734,14 @@ def from_array(cls, array, wcs=None, default_format=None):
 
         _validate_format('default_format', default_format)
 
-        array = np.atleast_2d(array)
+        # Windows systems ('nt') cannot close a file while there are any variables pointing
+        # to data within the opened file. Therefore we have to copy the entire array from
+        # the opened file. In other, more permissive operating systems, pointing to the
+        # file data is ok.
+        if os.name == 'nt':
+            array = np.copy(array)
+        else:
+            array = np.atleast_2d(array)
```

Comment:
I think we should call `atleast_2d` for both the Windows and non-Windows codepaths here. To be honest it's hard for me to see a case where a 1D array can be processed meaningfully as an image, but making sure the array is 2D will prevent strange exceptions later on if someone tries to load one.

Reply:
Of course! 🤦‍♂️